source/Windows.Devices.Gpio/GpioController.cs

```diff
@@ -49,6 +54,14 @@ public sealed class Gpio​Controller : IGpioController
         /// <returns>The default GPIO controller for the system, or null if the system has no GPIO controller.</returns>
         public static Gpio​Controller GetDefault()
         {
+            if (s_instance == null)
+            {
+                lock (_syncLock)
+                {
+                    s_instance = new Gpio​Controller();
```

Comment:
You have to check for null here again, because between the first check and the lock(), another thread might have created the instance